### ISSUE TITLE
(psycopg2.ProgrammingError) can't adapt type 'WKTElement'

### ISSUE THREAD

**Issue by GerbenJdeBoer** [geoalchemy/geoalchemy2#272]
At the transition from release 0.7.0 to 0.8.* the following error was introduced by sqlalchemy code that worked previously.
(psycopg2.ProgrammingError) can't adapt type 'WKTElement'

SQLAlchemy==1.3.17
psycopg2==2.7.3.2
python=3.6

**Comment by adrien-berchet** [comment]
Hi @GerbenJdeBoer , thanks for the report.
Could you please provide a small example giving you this error please? I am quite surprised  by this one.

**Comment by GerbenJdeBoer** [comment]
Some relevant code snippets from my application:
```
from shapely.geometry import shape
from geoalchemy2 import WKTElement, func

class Area(Base):
    """Area polygon."""

    __tablename__ = 'area'

    id = Column(
        Integer,
        Sequence('area_of_interest_id_seq'),
        primary_key=True)
    geom = Column(Geography(geometry_type='MULTIPOLYGON', srid=4326))

with self.path.open('r', encoding="utf8") as file:
    data = json.load(file)

for feature in data['features']:
    area = Area()
    area.geom = func.ST_Multi(func.ST_Intersection(
        func.ST_MakeValid(
            WKTElement(shape(feature['geometry']).wkt, 4326)),
        WKTElement((
            'POLYGON((-179.99 -89.99, -179.99 89.99, '
            '179.99 89.99, 179.99 -89.99, -179.99 -89.99))'),
            4326)))

```

**Comment by adrien-berchet** [comment]
Ok thanks
The problem here comes from `ST_MakeValid` which was not declared in the known functions of `geoalchemy2`. The release `0.8.3` should fix this.
Feel free to reopen if your issue is not fixed by `0.8.3`.
